smpte.js

```diff
@@ -456,7 +456,7 @@ function insertNormativeReferences(docMetadata) {
   /* style URLs */
 
   for(const u of sec.querySelectorAll("ul a")) {
-    u.parentNode.insertBefore(document.createTextNode("url:\u00a0"), u);
+    u.parentNode.insertBefore(document.createElement("br"), u);
```

Comment:
`url:` is the recommended means of signaling URLs in references

Reply:
Added back in on https://github.com/SMPTE/html-pub/pull/336/commits/86b5a4a7d0096b0b508292084ab958652978e50c